pkg/volume/testing/OWNERS

```diff
@@ -1,14 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
+- sig-storage-approvers
```

Comment:
Should we let this bubble up to pkg/volume/OWNERS?

Reply:
Sure.